ui/resizable.js

```diff
@@ -1072,9 +1074,9 @@ $.ui.plugin.add("resizable", "ghost", {
 				margin: 0,
 				left: 0,
 				top: 0
-			})
-			.addClass("ui-resizable-ghost")
-			.addClass(typeof o.ghost === "string" ? o.ghost : "");
+			});
```

Comment:
This now has a single method call, but has line break formatting for a multi-call statement. Should probably just chain with the previous line anyway.